src/CodeCoverage/Generators/AbstractGenerator.php

```diff
@@ -96,31 +96,13 @@ protected function getSourceIterator()
 			? new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($this->source))
 			: new \ArrayIterator([new \SplFileInfo($this->source)]);
 
-		return new AcceptIterator($iterator, $this->acceptFiles);
+		return new \CallbackFilterIterator($iterator, function (\SplFileInfo $file) {
+			return substr($file->getBasename(), 0, 1) !== '.'  // . or .. or .gitignore
```

Comment:
`$file->getBasename()[0]` should work fine assuming `getBasename()` cannot be empty